docs/UserGuide.md

```diff
@@ -3,27 +3,49 @@
 ## Introduction
 
 WerkIt! is an application for managing workout routines,
-optimized for use via a Command Line Interface (CLI).
+optimized for use via a Command Line Interface (CLI). This application is for you if you wish to
+have an application to keep track of your workouts and have a system to store your workout routines.
```

Comment:
i feel like we should not because, for beginners, in my mindset it is like having the need to teach them on how to carry out each exercise but in our application, we are not doing so

Reply:
perhaps for our future enhancement :P but yes i guess our UG will be for the current features we have! thanks!